tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -223,6 +223,23 @@ function xyz(
     );
 ",
             ),
+            'test half-multiline function gets full-multiline' => array(
+                <<<'EXPECTED'
```

Comment:
Are you ok with HEREDOC here? I find it is nicer to look at because the indention is no longer broken.